packages/hardhat-ovm/tslint-to-eslint-config.log

```diff
@@ -0,0 +1,21 @@
+7 ESLint rules behave differently from their TSLint counterparts:
+  * @typescript-eslint/no-unused-expressions:
+    - The TSLint optional config "allow-new" is the default ESLint behavior and will no longer be ignored.
+  * eqeqeq:
+    - Option "smart" allows for comparing two literal values, evaluating the value of typeof and null comparisons.
+  * no-invalid-this:
+    - Functions in methods will no longer be ignored.
+  * no-redeclare:
+    - ESLint does not support check-parameters.
+  * no-underscore-dangle:
+    - Leading or trailing underscores (_) on identifiers will now be forbidden.
+  * padded-blocks:
+    - ESLint's padded-blocks rule also bans a blank line before a closing brace.
+  * prefer-arrow/prefer-arrow-functions:
+    - ESLint does not support allowing standalone function declarations.
+    - ESLint does not support allowing named functions defined with the function keyword.
+
+2 rules are not known by tslint-to-eslint-config to have ESLint equivalents:
+  * tslint-to-eslint-config does not know the ESLint equivalent for TSLint's "no-focused-test".
+  * tslint-to-eslint-config does not know the ESLint equivalent for TSLint's "prettier".
+
```

Comment:
Is this auto-generated? Why does this package have it but not the others?
Do we need it, or can we put this info into the PR description?

Reply:
I ran the migration in every relevant subdir but realized the local `.eslintrc.js` could inherit from the base one and just have additional rules set as needed. I deleted the other `.log` files but missed this one. I can add this information to eh PR description.